### PR TITLE
Move wildfly to Java 8

### PR DESCRIPTION
**Pull request by ctomc**

### CODE REVIEW

**Comment by jamezp** [comment]
retest this please

File: feature-pack/src/main/resources/content/bin/appclient.conf [hunk 1]
```diff
@@ -42,7 +42,7 @@ fi
 # Specify options to pass to the Java VM.
 #
 if [ "x$JAVA_OPTS" = "x" ]; then
-   JAVA_OPTS="-Xms64m -Xmx512m -XX:MaxPermSize=256m -Djava.net.preferIPv4Stack=true"
```
Comment (dmlloyd) [inline feature-pack/src/main/resources/content/bin/appclient.conf]: How sure are you that there is no 8.x JVM that still uses permgen?  Remember we don't just run on Oracle JVM.

**Comment by dmlloyd** [comment]
Should not be merged until the basic SM tests pass again.

File: feature-pack/src/main/resources/content/bin/appclient.conf (hunk 1 above)
Comment (ctomc) [inline feature-pack/src/main/resources/content/bin/appclient.conf]: Only Sun's hotspot JVM used permgen and other JVMs tolerated XX:*PermSize parameters but didn't use them.
I removed it here as with JDK9 jvm fails to boot at all if it is present.

**Comment by ctomc** [comment]
SM tests now pass ^^

**Pull request merged**